cmd/tink-worker/container/container.go

```diff
@@ -0,0 +1,111 @@
+package container
+
+import (
+	"context"
+	"path"
+
+	"github.com/docker/docker/api/types"
+	"github.com/docker/docker/api/types/container"
+	"github.com/docker/docker/client"
+	"github.com/pkg/errors"
+)
+
+type configData struct {
+	name           string
+	pid            string
+	env            []string
+	volumeBindings []string
+	cmd            []string
+	imageName      string
+	registry       string
+	captureLogs    bool
+}
+
+type removeOpts struct {
+	force   bool
+	links   bool
+	volumes bool
+}
+
+// create creates container and returns containerid on success else returns error.
+func create(ctx context.Context, cli client.ContainerAPIClient, data configData) (string, error) {
+	config := &container.Config{
+		Image:        path.Join(data.registry, data.imageName),
```

Comment:
might want to have some data validation on `data`. for example, if we don't pass `data.registry` or `data.imageName` we can still create a container but we are almost certainly going to have a problem when we run `func start()`